pyanaconda/payload/dnf/payload.py

```diff
@@ -369,26 +369,6 @@ def get_addon_repo(self, repo_id):
 
         return repo
 
-    def _add_repo_to_dnf_and_ks(self, ksrepo):
-        """Add an enabled repo to dnf and kickstart repo lists.
-
-        Add the repo given by the pykickstart Repo object ksrepo to the
-        system.
-
-        Duplicate repos will not raise an error.  They should just silently
-        take the place of the previous value.
```

Comment:
This is not exactly documented afterwards, but I guess you plan to refactor that further?

Reply:
This is handled and documented in the `add_repository` method of the `DNFManager` class.